examples/megacli.cpp

```diff
@@ -3316,6 +3316,40 @@ void exec_cp(autocomplete::ACState& s)
 
 void exec_du(autocomplete::ACState& s)
 {
+    if (s.extractflag("-all"))
+    {
+        bool breakdown = s.extractflag("-breakdown");
+        struct counters { m_off_t filecount = 0, filebytes = 0, fileversioncount = 0, fileversionbytes = 0, foldercount = 0; };
+        map<Node*, counters> values;
+        for (auto& n : client->nodes)
+        {
+            auto& c = values[breakdown ? n.second->firstancestor() : nullptr];
+            switch (n.second->type)
```

Comment:
We get a warning here because not all enum values are handled. You need to add a default case that has `assert(false)` or something like that.

Reply:
will update